pkg/operator/status/condition.go

```diff
@@ -38,7 +40,9 @@ func unionCondition(conditionType string, defaultConditionState operatorv1.Condi
 		return OperatorConditionToClusterOperatorCondition(unionedCondition)
 	}
 
-	if len(badConditions) == 0 {
+	latestTransitionTimeBadConditions := latestTransitionTime(badConditions)
+	if len(badConditions) == 0 || latestTransitionTimeBadConditions.Add(time.Minute).After(time.Now()) {
```

Comment:
No.  This could cause flapping when we've been failing for 10 minutes and suddenly something else starts failing.  Add a test case that would have caught that case.

For all the bad conditions, we want to see if any of them transitioned more than one minute ago.  I think this wants the earliest transition time of bad conditions.

The reason this uses the latest is that if the reason changes (and it would), the time of the last transition is when the reason changed, not when the status changed.